### PR TITLE
updated README file with sample code.

### PR DESCRIPTION
**Pull request by nithyaganeshng**
updated README file with introduction and sample code.

**Solution:**
Used library A to complete scenario X with quality and style.

**Validation:**
Please include here how it was verified that the PR works as intended.

**Type of change:**
- [ ] Feature work
- [ ] Bug fix
- [x] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,99 @@
-# Project
+# Microsoft Entra Wallet Library
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+## Introduction
+The Microsoft Entra Wallet Library for iOS gives your app the ability to begin using the Microsoft Entra Verified Id platform by supporting the issuance and presentation of Verified Ids in accordance with OpenID Connect, Presentation Exchange, Verifiable Credentials, and more up and coming industry standards.
```
Comment (symorton) [inline README.md]: nit: Android

File: README.md [hunk 2]
```diff
@@ -1,14 +1,99 @@
...
 
-As the maintainer of this project, please make a few updates:
+---
+## Installation
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Add to your app's build.gradle to add Wallet Library as a dependency:
```
Comment (symorton) [inline README.md:9]: installation instructions are all on one line. Maybe put the dependencies info in a code snippet?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,99 @@
...
+```kotlin
+// Encode a VerifiedId into String.
+val encodedVerifiedIdString = verifiedIdClient.encode(verifiedId)
+// Decode a VerifiedId from String.
+val verifiedId = verifiedIdClient.decodeVerifiedId(encodedVerifiedIdString)
+```
+
+## Documentation
+
+* [External Architecture](Docs/LibraryArchitecture.md)
```
Comment (symorton) [inline README.md]: Add Library Arch diagram from iOS (should be the same for both platforms anyways) repo to this one to make this link work correctly.

**Review by symorton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nithyaganeshng) [reply README.md]: copypasta, I will fix it.

File: README.md (hunk 3 above)
Comment (nithyaganeshng) [reply README.md]: I actually copied from iOS.

**Review by symorton — APPROVED** [review]

**Pull request merged**